### ISSUE TITLE
Skip waypoints if occupied

### ISSUE THREAD

**Issue by tonynajjar** [open-navigation/opennav_coverage#68]
I plan on using opennav_coverage in the future but haven't yet so I can't answer this question myself: Is there any functionality to skip waypoints if they are occupied (for too long). If not then this is an issue to implement that (opt-in).

**Comment by SteveMacenski** [comment]
No, but there are options to specify rows to use - so you could replan excluding it and any completed rows -- so in spirit but not as a "here's a parameter to do it"

**Comment by tonynajjar** [comment]
Excluding a complete row is a bit excessive and impacts the coverage quite a lot. Alright, already good to know that it's not possible. Will come back with a design proposal when I come back to this

**Comment by SteveMacenski** [comment]
This is mostly a F2C thing: its really more meant for static fields that you know about the obstacles in to begin with. However, adding another module that uses some F2C and some custom (like the Row Coverage server) that does more active choices would be welcome here :-)
